list-your-project/how-to-write-dimension-adapter.md

```diff
@@ -0,0 +1,695 @@
+# Building Dimension Adapters
+
+This guide will help you create adapters for DefiLlama's various dashboards, including [fees](https://defillama.com/fees), [volumes](https://defillama.com/dexs), [aggregators](https://defillama.com/aggregators), [derivatives](https://defillama.com/derivatives), [Bridge Aggregators](https://defillama.com/bridge-aggregators), [Options](https://defillama.com/options), and others.
+
+## What is an Adapter?
+
+An adapter is some code that:
+
+1. Collects data on a protocol by calling some endpoints or making blockchain calls
+2. Computes a response and returns it
+
+It's a TypeScript file that exports an async function which takes start and end timestamps (and block numbers) and returns an object with metrics for that time range.
```

Comment:
better to explain various params passed as part of fetchOptions? or state that the function takes time or block range as input and returns data for that period?